tests/telemetry_derived/clients_daily_v7/test_aggregation/expect.yaml

```diff
@@ -0,0 +1,152 @@
+  histogram_parent_devtools_fontinspector_opened_count_sum: 2
+  histogram_parent_devtools_inspector_opened_count_sum: 2
+  histogram_parent_devtools_jsbrowserdebugger_opened_count_sum: 2
+  histogram_parent_devtools_jsdebugger_opened_count_sum: 2
+  histogram_parent_devtools_jsprofiler_opened_count_sum: 2
+  histogram_parent_devtools_layoutview_opened_count_sum: 2
+  histogram_parent_devtools_memory_opened_count_sum: 2
+  histogram_parent_devtools_menu_eyedropper_opened_count_sum: 2
+  histogram_parent_devtools_netmonitor_opened_count_sum: 2
+  histogram_parent_devtools_options_opened_count_sum: 2
+  histogram_parent_devtools_paintflashing_opened_count_sum: 2
+  histogram_parent_devtools_picker_eyedropper_opened_count_sum: 2
+  histogram_parent_devtools_responsive_opened_count_sum: 2
+  histogram_parent_devtools_ruleview_opened_count_sum: 2
+  histogram_parent_devtools_scratchpad_opened_count_sum: 2
+  histogram_parent_devtools_scratchpad_window_opened_count_sum: 2
+  histogram_parent_devtools_shadereditor_opened_count_sum: 2
+  histogram_parent_devtools_storage_opened_count_sum: 2
+  histogram_parent_devtools_styleeditor_opened_count_sum: 2
+  histogram_parent_devtools_webaudioeditor_opened_count_sum: 2
+  histogram_parent_devtools_webconsole_opened_count_sum: 2
+  histogram_parent_devtools_webide_opened_count_sum: 2
+  install_year: 2
+  is_default_browser: true
+  is_wow64: true
+  locale: second
+  memory_mb: 2
+  normalized_channel: second
+  normalized_os_version: second
+  os: second
+  os_service_pack_major: 2
+  os_service_pack_minor: 2
+  os_version: second
+  pings_aggregated_by_this_row: 12
+  places_bookmarks_count_mean: 2
+  places_pages_count_mean: 2
+  plugin_hangs_sum: 2
+  plugins_infobar_allow_sum: 2
+  plugins_infobar_block_sum: 2
+  plugins_infobar_shown_sum: 2
+  plugins_notification_shown_sum: 2
+  previous_build_id: second
+  push_api_notify_sum: 2
+  sample_id: 0
+  sandbox_effective_content_process_level: 2
+  scalar_combined_webrtc_nicer_stun_retransmits_sum: 4
+  scalar_combined_webrtc_nicer_turn_401s_sum: 4
+  scalar_combined_webrtc_nicer_turn_403s_sum: 4
+  scalar_combined_webrtc_nicer_turn_438s_sum: 4
+  scalar_content_navigator_storage_estimate_count_sum: 2
+  scalar_content_navigator_storage_persist_count_sum: 2
+  scalar_parent_aushelper_websense_reg_version: second
+  scalar_parent_browser_engagement_max_concurrent_tab_count_max: 3
+  scalar_parent_browser_engagement_max_concurrent_window_count_max: 3
+  scalar_parent_browser_engagement_tab_open_event_count_sum: 2
+  scalar_parent_browser_engagement_total_uri_count_sum: 2
+  scalar_parent_browser_engagement_unfiltered_uri_count_sum: 2
+  scalar_parent_browser_engagement_unique_domains_count_max: 3
+  scalar_parent_browser_engagement_unique_domains_count_mean: 2
+  scalar_parent_browser_engagement_window_open_event_count_sum: 2
+  scalar_parent_devtools_accessibility_select_accessible_for_node_sum:
+    key_value: []
```

Comment:
`experiments`, `active_addons`, and `scalar_parent_devtools_accessibility_select_accessible_for_node_sum` are not currently tested, but that was already the case, so I would prefer to handle that in a separate PR